src/components/DocumentCard/DocumentCard.Props.ts

```diff
@@ -22,20 +28,53 @@ export interface IDocumentCardProps extends React.Props<DocumentCard> {
   onClickHref?: string;
 
   /**
-   * Optional class for document card
+   * Optional class for document card.
    */
   className?: string;
+
+  /**
+   * Hex color value of the line below the card, which should correspond to the document type.
+   * This should only be supplied when using the 'compact' card layout.
+   */
+  accentColor?: string;
+}
+
+export enum DocumentCardType {
+  /**
+   * Standard DocumentCard.
+   */
+  normal,
+  /**
+   * Compact layout. Displays the preview beside the details, rather than above.
+   */
+  compact
 }
 
 export interface IDocumentCardPreviewProps extends React.Props<DocumentCardPreview> {
   /**
    * One or more preview images to display.
    */
   previewImages: IDocumentCardPreviewImage[];
+
+  /**
+   * The format for the text that should display the number of overflow documents.
+   * such as '+2 more'
```

Comment:
Maybe this should be the type
(count) => string
And then example comment should be:

```typescript
(displayCount) => `=${ displayCount } more`;
```